src/common/options/mgr.yaml.in

```diff
@@ -103,6 +103,13 @@ options:
   services:
   - mgr
   with_legacy: true
+- name: mgr_max_pg_creating
```

Comment:
nit: I think that it makes more sense to both remove `mon_osd_max_creating_pgs`  and introduce `mgr_max_pg_creating` in the same commit _after_  removing the other user of it (commit order wise).